views/landing.handlebars

```diff
@@ -1,19 +1,125 @@
+        let city;
+        let state;
+        
+        // Try HTML5 geolocation.
+        if (navigator.geolocation) {
+          navigator.geolocation.getCurrentPosition(function(position) {
+            pos = {
+              lat: position.coords.latitude,
+              lng: position.coords.longitude
+            };
+            map.setCenter(pos);
+            map.setZoom(11);
+            geocoder.geocode({'location':pos},function(results,status){
+                console.log(results);
+                if(status === 'OK'){
+                    if(results[0]){
+                        const res = results[0].address_components;
+                        for(let i = 0; i<res.length; i++){
+                            for(let j = 0; j<res[i].types.length; j++){
+                                if(res[i].types[j] === 'sublocality' || res[i].types[j] === 'locality'){
+                                    city = res[i].long_name;
+                                }else if(res[i].types[j] === 'administrative_area_level_1'){
+                                    state = res[i].short_name;
+                                }
+                            }
+                        }
+                        console.log(city);
+                        console.log(state);
+                        getSellers(city,state,map,markers);
+                    }
+                }
+            });
+        });
+    }
+        
+        //get database city
+        
+}
+function getSellers(city, state,map,markers){
+    let latLngArr = [];
+    let namesArr = [];
+    let emailArr = [];
+    let idArr = [];
+    $.get("/api/sellers/city/" + city + "," + state, function(data){
+        if(data){
+            console.log(data);
+            for(let i = 0; i<data.length; i++){
+                const latlngObj = {lat: data[i].lat, lng: data[i].lng};
+                latLngArr.push(latlngObj);
+                namesArr.push(data[i].seller_name);
+                emailArr.push(data[i].email);
+                idArr.push(data[i].id);
+            }
+        genAddresses(latLngArr,namesArr,emailArr,idArr,map,markers);
+        }
 
-    </div>
+    });
+}
+function genAddresses(latLngArr,namesArr,emailArr,idArr,map,markers){
+    console.log(latLngArr);
+    console.log(namesArr);
+    console.log(emailArr);
+    console.log(idArr);
+    for(let i = 0; i<latLngArr.length; i++){
+        console.log(latLngArr[i]);
+        const geocoder = new google.maps.Geocoder();
+        geocoder.geocode({'location': {lat: parseFloat(latLngArr[i].lat), lng: parseFloat(latLngArr[i].lng)}}, function(results, status){
+            if(status === 'OK'){
+                if(results[0]){
+                    genMarkers(namesArr[i],emailArr[i],results[0].formatted_address,{lat: parseFloat(latLngArr[i].lat), lng: parseFloat(latLngArr[i].lng)},idArr[i],map,markers);
+                }
+            }
+        });
+    }
+}
+function genMarkers(name,email,address, latlng,id, map,markers){
```

Comment:
Code is fine but format is off where you have a space after some commas, but not all.